ext/RMagick/rmimage.c

```diff
@@ -4599,8 +4599,7 @@ Image_convolve_channel(int argc, VALUE *argv, VALUE self)
         rb_raise(rb_eArgError, "order must be non-zero and positive");
     }
 
-    ary = argv[1];
```

Comment:
@Watson1978 would it make sense to raise a ruby error here instead?

Reply:
rb_Array() will raise an exception if can't convert.
Without it, it just causes a SEGV.